tests/h/models/user_test.py

```diff
@@ -174,6 +174,19 @@ def test_userid_in_query_with_only_invalid_userid(db_session):
     assert result == []
 
 
+def test_rendered_name_returns_display_name_if_set(db_session):
+    fred = models.User(authority='hypothes.is',
+                       username='fred',
+                       display_name='Fred Perry')
+    assert fred.rendered_name == 'Fred Perry'
```

Comment:
I think if you use `factories` here to create the `User` then you won't have to specify `authority` (which isn't relevant to this test)